benches/Cargo.toml

```diff
@@ -7,7 +7,7 @@ publish = false
 license = "MIT OR Apache-2.0"
 
 [dev-dependencies]
-glam = "0.20"
+glam = "0.21.2"
```

Comment:
Preferentially use "0.21" over "0.21.2". No reason to pin to a specific patch version.

Reply:
Thanks. I never know what to do about these, and I did look at the other imports, and they didn't have a patch number also, but the upgrade was for `0.21.1`... But now it is fixed!